.github/labeler.yml

```diff
@@ -0,0 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Configuration for .github/workflows/label-pr.yml
+# This workflow and its configuration can be deleted once the container reconciliation feature branch is merged.
```

Comment:
Once this workflow is set up, we'll likely add further labeling rules.  When the feature branch is merged, I'd keep the labeler and only delete the specific rule.

```suggestion
# This rule can be deleted once the container reconciliation feature branch is merged.
```

Reply:
On the same note, how about adding this workflow on `master`, and starting the feature branch from there?